L1TriggerConfig/L1TConfigProducers/test/BuildFile.xml

```diff
@@ -0,0 +1,7 @@
+<architecture name="(slc|cc).*_amd64_.*">
```

Comment:
@panoskatsoulis , please just use `<architecture name="_amd64_">` as it will cover all of our `amd64` archs